sdk/python/foundation-models/system/import/import_model_into_registry.ipynb

```diff
@@ -505,9 +508,9 @@
  "metadata": {
   "celltoolbar": "Tags",
   "kernelspec": {
-   "display_name": "Python 3.10 - SDK V2",
+   "display_name": "v2",
    "language": "python",
-   "name": "python310-sdkv2"
+   "name": "python3"
```

Comment:
Is this change expected? Probably got updated with local terminal settings?

Reply:
Should we also make sure to set kernel for finetuning and evaluation notebooks as well?

Reply:
Yes. Didn't make any explicit change. Thanks for catching. Will revert it.

Reply:
Done.